README.md

```diff
@@ -0,0 +1,35 @@
+# Aserto shared Type-check/Lint/Format configs
+
+## Testing changes locally
+
+To quickly check that configs are valid, don't have syntax errors, etc. run the checks on this repo.
+
+### Type-checking
+```yarn tsc```
```

Comment:
this is expected to fail with:

```shell
error TS18003: No inputs were found in config file '/home/piotr_aserto_com/code/github/aserto/ts-linting-configs/tsconfig.json'. Specified 'include' paths were '["src"]' and 'exclude' paths were '[]'.

Probably useful to call that out here.
```

Reply:
Alternatively, we could add an empty `src/index.d.ts`. WDYT

Reply:
sure works for me